DEVELOPMENT.md

```diff
@@ -93,25 +119,66 @@ After which, you can rerun a `px auth login` to authenticate with the dev cloud
 Make sure to `px delete --clobber` if running a prior instance of Vizier pointing to another cloud instance.
 
 ## Vizier
+Vizier is Pixie’s data collector that runs on each cluster. It is responsible
+for query execution and managing PEMs.
+
+### Getting Started
+Verify that you can build Pixie Vizier and run the unit tests.
+
+The following will build Pixie Vizier and then run the unit tests for the UDF
+(User Defined Function) math operations. These UDFs will be available directly off
+the px object inside a PxL script. If you are running the development environment via
+Docker, the following should be run inside the Docker container.
+
+```
+bazel test //src/carnot/funcs/builtins:math_ops_test --test_output=errors -j 2
+```
```

Comment:
Things might more around so I'm not sure we want to list a specific test. We can also just run:

`bazel test //...` 

to make sure all unit tests pass. 

You can use -j $(nproc) which will allow this to scale to the number of CPUs you have available.